docs/azure_arc_jumpstart/azure_arc_k8s/aks_stack_hci/aks_hci_powershell/_index.md

```diff
@@ -0,0 +1,194 @@
+    - **resourceGroup:** resource group to connect your Azure Arc enabled Kubernetes.
+    - **location:** Azure region to connect your Azure Arc enabled Kubernetes.
+    - **subscriptionId:** subscription to connect your Azure Arc enabled Kubernetes.
+    - **appId:** the appID of the service principal created previously.
+    - **password:** the password of the service principal created.
+    - **tenant:** your tenantID.
+
+![Screenshot showing the AKS on HCI deployment script](./02.png)
+
+* As an example: 
+    - **vipPoolStart:** "192.168.0.150"
+    - **vipPoolEnd:** "192.168.0.250"
+    - **imageDir:** "C:\AKS-HCI\Images"
+    - **cloudConfigLocation:** "C:\AKS-HCI\Config"
+    - **vnetName:** "InternalNAT"
+    - **clusterName:** "archcidemo"
+    - **controlPlaneNodeCount:** 1
+    - **linuxNodeCount:** 1
+    - **windowsNodeCount:** 0
+    - **resourceGroup:** "Arc-AKS-HCI-Demo"
+    - **location:** westeurope
+    - **subscriptionId:** "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX"
+    - **appId:** "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX"
+    - **password:** "XXXXXXXXXX"
+    - **tenant:** "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX"
+
+* Note, that the script will deploy a simple DHCP based cluster on your Azure Stack HCI, there are additional optional parameters that you could use to customize the deployment to your own environment as described [here] (https://docs.microsoft.com/en-us/azure-stack/aks-hci/setup-powershell). 
+
+* To run the script open PowerShell as an administrator, navigate to the [script folder](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/aks_stack_hci/powershell/) and run: 
+
+  ```powershell
+  .\aks_hci_deploy.ps1
+  ```
+![Screenshot showing the AKS on HCI deployment script running](./03.png)
+
+![Screenshot showing the AKS on HCI deployment script running](./04.png)
+
+  > **Note: the script may take around 30 minutes to run**
+
+* Once the script has completed its run 
+
+![Screenshot showing the script output](./05.png)
+
+* You should also see the new AKS cluster on Windows Admin Center. 
+
+![Screenshot showing the AKS on HCI deployment Windows Admin Center](./06.png)
+
+* Once the script run has finished, the AKS cluster on HCI will be projected as a new Azure Arc cluster resource.
+
+![Screenshot showing Arc Enabled Kubernetes in RG](./07.png)
+
+![Screenshot showing Arc Enabled Kubernetes](./08.png)
+
+## Delete the deployment
+
+The most straightforward way is to delete the Azure Arc cluster resource via the Azure Portal, just select the cluster and delete it.
+
+![Screenshot showing how to delete Azure Arc enabled Kubernetes resource](./09.png)
+
+If you want to delete the AKS cluster on HCI run the below command, this will delete all of your AKS clusters on HCI if any and the Azure Kubernetes Service host. It will also uninstall the Azure Kubernetes Service on Azure Stack HCI agents and services from the nodes. 
+
```

Comment:
```suggestion
To delete the AKS cluster on HCI run the below command. This will delete all of your AKS clusters on HCI (if any) and the Azure Kubernetes Service host as well as uninstall the Azure Kubernetes Service on Azure Stack HCI agents and services from the nodes. 

```